### PR TITLE
Corrections to up-ga and up-logs sessions

### PR DESCRIPTION
**Pull request by CRMGB**

### CODE REVIEW

File: docs/up-logs/sessions/v1.md [hunk 1]
```diff
@@ -11,8 +11,27 @@
 <div id="value-display"></div>
 <strong>Description:</strong>
 <div class="tile-1" style="text-align:center; color:black">
-Total sessions for the landing page of a work, as determined by processing the Access Logs of the website. A Session is a group of visits made by the same user within a continuous time frame, and ends after 30 minutes of inactivity; if a single user keeps interacting with the website within this time frame, multiple visits to the same work will be counted as one session.
+Total views of the landing page of a work, as determined by processing the access logs (i.e. webserver logs) of the website. Multiple views of the same landing page by the same user within a rolling 30 minute window are counted as one.
```
Comment (fradeve) [inline docs/up-logs/sessions/v1.md:14]: This is phrased differently from `up-ga/sessions`... can we use the same terminology?

File: site/up-logs/sessions/v1/index.html [hunk 2]
```diff
@@ -897,8 +897,25 @@ <h1 id="up-logs-sessions-measure">Up-logs Sessions Measure</h1>
 <div id="value-display"></div>
 <p><strong>Description:</strong></p>
 <div class="tile-1" style="text-align:center; color:black">
-Total sessions for the landing page of a work, as determined by processing the Access Logs of the website. A Session is a group of visits made by the same user within a continuous time frame, and ends after 30 minutes of inactivity; if a single user keeps interacting with the website within this time frame, multiple visits to the same work will be counted as one session.
+Total views of the landing page of a work, as determined by processing the access logs (i.e. webserver logs) of the website. Multiple views of the same landing page by the same user within a rolling 30 minute window are counted as one.
```
Comment (fradeve) [inline site/up-logs/sessions/v1/index.html:900]: same as comment above

**Review by fradeve — CHANGES_REQUESTED** [review]

**Review by fradeve — APPROVED** [review]

File: docs/up-ga/sessions/v1.md [hunk 3]
```diff
@@ -9,10 +9,66 @@
 # Up-ga Sessions Measure
 
 <div id="value-display"></div>
+<br>
 <strong>Description:</strong>
 <div class="tile-1" style="text-align:center; color:black">
-Total sessions for the landing page of a work, as determined by Google Analytics. A Session is a group of visits made by the same user within a continuous time frame, and ends after 30 minutes of inactivity; if a single user keeps interacting with the website within this time frame, multiple visits to the same work will be counted as one session.
+Total views of the landing page of a work, as determined by Google Analytics (i.e. webserver logs) of the website. To count multiple views of the same landing page by the same user, there must be a gap of at least 30 minutes from the last view of that landing page, even if that previous view was not counted.
```
Comment (rowan08) [inline docs/up-ga/sessions/v1.md:15]: I think it's best to remove "i.e. webserver logs" - Google Analytics and webserver logs are two different things. Google Analytics uses JavaScript to measure page visits and page interactions.

File: docs/up-ga/downloads/v1.md [hunk 4]
```diff
@@ -11,13 +11,68 @@
 <div id="value-display"></div>
 <strong>Description:</strong>
 <div class="tile-1" style="text-align:center; color:black">
-    No data available for the specified measure.
+  Total downloads of the landing page of a work, as determined by Google Analytics of the website. To count multiple downloads of the same landing page by the same user, there must be a gap of at least 30 minutes from the last download of that landing page, even if that previous download was not counted.
```
Comment (fradeve) [inline docs/up-ga/downloads/v1.md:14]: remove `of the website`.

A landing page is not downloaded, a file is; change to:
`To count multiple downloads of the same file by the same user`

File: docs/up-logs/downloads/v1.md [hunk 5]
```diff
@@ -11,13 +11,68 @@
 <div id="value-display"></div>
 <strong>Description:</strong>
 <div class="tile-1" style="text-align:center; color:black">
-Total number of times a work has been downloaded, as determined by processing the Access Logs of the website. Multiple downloads of the same work that occur over a single session will be counted as one download. A Session is a group of visits made by the same user within a continuous time frame, and ends after 30 minutes of inactivity.
+Total number of times a work has been downloaded, as determined by processing the Access Logs of the website. To count multiple downloads of the same landing page by the same user, there must be a gap of 30 minutes from the last counted download.
```
Comment (fradeve) [inline docs/up-logs/downloads/v1.md:14]: same points on downloads as in previous comment

**Review by fradeve — CHANGES_REQUESTED** [review]

File: docs/up-ga/reads/v1.md [hunk 6]
```diff
@@ -11,8 +11,64 @@
...
+<strong>
+For Example:
+</strong>
+
+<div style="text-align: center;">
+<table>
+    <tr>
+        <th>Time of visit</th>
+        <th>Minutes from last visit</th>
+        <th>Views counted</th>
```
Comment (rowan08) [inline docs/up-ga/reads/v1.md:27]: Reads counted

File: docs/up-ga/downloads/v1.md [hunk 7]
```diff
@@ -11,13 +11,68 @@
...
+<strong>
+For Example:
+</strong>
+
+<div  style="text-align:center; color:black">
+<table>
+    <tr>
+        <th>Time of download</th>
+        <th>Minutes from last download</th>
+        <th>downloads counted</th>
```
Comment (rowan08) [inline docs/up-ga/downloads/v1.md:26]: For consistency, Downloads should be capitalized

File: docs/up-logs/downloads/v1.md [hunk 8]
```diff
@@ -11,13 +11,68 @@
...
+<strong>
+For Example:
+</strong>
+
+<div style="text-align: center;">
+    <table style="margin: 0 auto;">
+        <tr>
+          <th>Time of download</th>
+          <th>Minutes from last download</th>
+          <th>downloads counted</th>
```
Comment (rowan08) [inline docs/up-logs/downloads/v1.md:26]: For consistency, Downloads should be capitalized

File: docs/up-logs/reads/v1.md [hunk 9]
```diff
@@ -11,8 +11,64 @@
...
+<strong>
+For Example:
+</strong>
+
+<div style="text-align: center;">
+    <table style="margin: 0 auto;">
+        <tr>
+          <th>Time of visit</th>
+          <th>Minutes from last visit</th>
+          <th>Views counted</th>
```
Comment (rowan08) [inline docs/up-logs/reads/v1.md:26]: Reads Counted

File: docs/up-ga/downloads/v1.md [hunk 10]
```diff
@@ -11,13 +11,68 @@
 <div id="value-display"></div>
 <strong>Description:</strong>
 <div class="tile-1" style="text-align:center; color:black">
-    No data available for the specified measure.
+  Total downloads of the landing page of a work, as determined by Google Analytics. To count multiple downloads of the same file by the same user, there must be a gap of at least 30 minutes from the last download of that landing page, even if that previous download was not counted.
```
Comment (rowan08) [inline docs/up-ga/downloads/v1.md:14]: Again, landing pages don't get downloaded

File: docs/up-ga/downloads/v1.md (hunk 7 above)
Comment (fradeve) [reply docs/up-ga/downloads/v1.md:26]: `<th>Downloads counted</th>`

File: docs/up-logs/downloads/v1.md (hunk 8 above)
Comment (fradeve) [reply docs/up-logs/downloads/v1.md:26]: `<th>Downloads counted</th>`

File: docs/up-ga/reads/v1.md [hunk 11]
```diff
@@ -11,13 +11,69 @@
 <div id="value-display"></div>
 <strong>Description:</strong>
 <div class="tile-1" style="text-align:center; color:black">
-No data available for the specified measure.
+  Total reads on pages where the work can be read, as determined by Google Analytics. To count multiple reads on the same page by the same user, there must be a gap of at least 30 minutes from the last read of that page, even if that previous read was not counted.
```
Comment (rowan08) [inline docs/up-ga/reads/v1.md:14]: I really think this should be "Total views on pages where the work can be read".

Am happy to discuss tomorrow

**Pull request merged**